odoo/tests/common.py

```diff
@@ -702,12 +702,22 @@ def executable(self):
                     return bin_
 
         elif system == 'Windows':
-            # TODO: handle windows platform: https://stackoverflow.com/a/40674915
-            pass
+            bins = [
+                '%ProgramFiles%\Google\Chrome\Application\chrome.exe',
+                '%ProgramFiles(x86)%\Google\Chrome\Application\chrome.exe',
+                '%LocalAppData%\Google\Chrome\Application\chrome.exe',
+            ]
+            for bin_ in bins:
+                bin_ = os.path.expandvars(bin_)
+                if os.path.exists(bin_):
+                    return bin_
 
         raise unittest.SkipTest("Chrome executable not found")
 
     def _spawn_chrome(self, cmd):
+        if os.name == 'nt':
+            proc = subprocess.Popen(cmd)
```

Comment:
Since we replace the chrome process' `stdout` by `devnull` in the fork case, this should do the same.

Also given we don't do anything special for the case of darwin (macos), this case could be `os.name == 'nt' or platform.system()  == 'Darwin'`.

Alternatively, and probably more simply, the condition here could be `platform.system() in ('Darwin', 'Windows')` and we assume everything else is "regular posix" on which we need to reset the rlimit.

Even more alternatively (though probably only on master), we could move everything to `subprocess.Popen` and use `preexec_fn` to unset rlimit.

Reply:
> Since we replace the chrome process' `stdout` by `devnull` in the fork case, this should do the same.

OK, I considered it for subprocess

> Also given we don't do anything special for the case of darwin (macos), this case could be `os.name == 'nt' or platform.system() == 'Darwin'`.
> 
> Alternatively, and probably more simply, the condition here could be `platform.system() in ('Darwin', 'Windows')` and we assume everything else is "regular posix" on which we need to reset the rlimit.

I've checked tests run on macOS and fork case works well for this platform. 

> Even more alternatively (though probably only on master), we could move everything to `subprocess.Popen` and use `preexec_fn` to unset rlimit.

I don't have enough experience in forwardport process. Should I create separate PR for master after this PR forwarding or before?

Reply:
> I've checked tests run on macOS and fork case works well for this platform.

Of course, my point is that the reason for using `fork` doesn't exist for macOS, because we *don't* need to call `resource.setrlimit`. So if we're going to fix this for windows using `subprocess.Popen`, we might as well use that for darwin as well. This will allow somewhat better exercising the feature, and will simplify the code (as it'll remove a branch in the `fork` path).

> I don't have enough experience in forwardport process. Should I create separate PR for master after this PR forwarding or before?

No, the process is that once a PR is merged, the bots will try to reapply it on the next branch in the sequence (e.g. 14.0) and will keep doing that as long as CI succeeds.

Reply:
I suggest to stay on compact diff for 13.0-15.0 and propose more aggressive change for master in https://github.com/odoo/odoo/pull/83209